AWS/DirectoryInsights/get-jcdirectoryinsights.py

```diff
@@ -0,0 +1,76 @@
+import requests, datetime, json, boto3, os, gzip
+
+def jc_directoryinsights(event, context):
+    jcapikey = os.environ['JCAPIKEY']
```

Comment:
Can you add a try except here for all the env variables? If you get a key error then raise a keyerror exception that contains the missing env variable in it